apps/test-can/src/main_old.cpp

```diff
@@ -0,0 +1,71 @@
+#include "CANBus.h"
```

Comment:
This is the old test-can app, leaving it here for now but will delete it once the new test app is fleshed out